src/cc_catalog_airflow/dags/util/loader/sql.py

```diff
@@ -390,3 +391,59 @@ def update_flickr_sub_providers(
     Drop the temporary table
     """
     postgres.run(f'DROP TABLE public.{temp_table};')
+
+
+def update_europeana_sub_providers(
+  postgres_conn_id,
+  image_table=IMAGE_TABLE_NAME,
+  default_provider=prov.EUROPEANA_DEFAULT_PROVIDER,
+  sub_providers=prov.EUROPEANA_SUB_PROVIDERS
+):
+    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+
+    select_query = dedent(
+        f'''
+        SELECT
+        {col.FOREIGN_ID} AS foreign_id,
+        {col.META_DATA} ->> 'dataProvider' AS data_providers
+        FROM {image_table}
+        WHERE {col.PROVIDER} = '{default_provider}';
+        '''
+    )
```

Comment:
I think we should create a temporary table as in the Flickr version of this and join with it to further restrict the results.  The reason is that over time (and relatively soon) Europeana will be growing significantly in volume, and I'd rather not pull 20 million results down over the network to modify 10,000 of them.

It's possible to join on array membership using the `?` operator in PostgreSQL:
```SQL
CREATE TABLE public.eur_sources (data_provider varchar(108), source varchar(108));
INSERT INTO eur_sources
  (data_provider, source) 
VALUES
  ('Wellcome Collection', 'wellcome_collection');

SELECT L.identifier, L.source, L.meta_data, R.data_provider, R.source
FROM (
  SELECT identifier, source, meta_data 
  FROM image
  WHERE provider='europeana'
) L INNER JOIN
  eur_sources R ON
  L.meta_data->'dataProvider' ? R.data_provider;
```

Reply:
This select will only pull down results with 'Wellcome Collection' in the `dataProvider` field of `meta_data`.